src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -417,20 +417,20 @@ public void execute(StartupChecksOptions options) throws StartupException
         }
     };
 
-    public static final StartupCheck initSigarLibrary = new StartupCheck()
+    public static final StartupCheck checkProcessEnvironment = new StartupCheck()
     {
         @Override
         public void execute(StartupChecksOptions options)
         {
             if (options.isDisabled(getStartupCheckType()))
                 return;
-            SigarLibrary.instance.warnIfRunningInDegradedMode();
+            new SystemInfo().isDegraded().ifPresent( s -> logger.warn("Cassandra server running in degraded mode. " + s));
```

Comment:
@Claudenw was not the deal to keep info log there? https://github.com/apache/cassandra/pull/2842#discussion_r1441646580